app/layout.tsx

```diff
@@ -18,15 +18,18 @@ export default function RootLayout({
   return (
     <html lang="en">
       <head>
+        <ColorSchemeScript />
       </head>
       <body className="w-10/12 flex flex-col m-auto">
-        <header>
-          <LinktaLogoWithText />
-        </header>
-        <main className="flex flex-col my-3">{children}</main>
-        <footer className="sm:flex flex-col">
-          <Footer />
-        </footer>
+        <MantineProvider>
+          <header>
+            <LinktaLogoWithText />
```

Comment:
Just curious, is it intentional to keep the LinktaLogoWithText in both `layout.tsx` and main `app/page.tsx`?

Reply:
Good question - I didn't address this in this PR. I think it would be a better discussion for the main styling PR #23. These are my suggested updates to that PR.

Reply:
This was a bit of a hacky workaround to keep the header in all pages on the desktop application, but still have the header appear on only the landing page for the mobile app.